### ISSUE TITLE
[Accessibility] Device Dependent handlers are used. Tooltips are not accessible with keyboard.

### ISSUE THREAD

**Issue by puneet-ahuja** [vega/vega-lite#9197]
Tooltips in the charts are not accessible with the help of a keyboard.

**Steps to replicate:**

Open the [Codesandbox Link](https://codesandbox.io/p/sandbox/tooltip-not-accessible-with-keyboard-yh3ldj?file=%2Findex.html%3A63%2C8).

The above link opens an interactive multi-line chart.
We can access the tooltips on the lines using mouse hover but not with keyboards.

**Comment by domoritz** [comment]
This is an issue that needs to be addressed in Vega. The challenge is that we render to canvas as well and so there is no dom element that can listen to events. We probably need to use something like https://www.frank.computer/data-navigator/ to address that. 

Please file an issue with Vega with some of the details of the handlers and how we could change them.
